maps/tether/tether-03-surface3.dmm

```diff
@@ -23598,12 +23601,6 @@
 	},
 /turf/simulated/floor/plating,
 /area/tether/surfacebase/public_garden_three)
-"Ym" = (
-/obj/machinery/door/airlock/glass{
-	name = "The Game Den"
-	},
-/turf/simulated/floor/tiled/steel_grid,
-/area/tether/surfacebase/outside/outside3)
```

Comment:
`/area/tether/surfacebase/outside/outside3`
This is why it wasn't working. Change it to be either the bar's area or the hallway's area.

Reply:
Trust me, I'm a mapper. :3